src/app/views/authentication/profile/Profile.tsx

```diff
@@ -35,6 +36,8 @@ import { translateMessage } from '../../../utils/translate-messages';
 import { classNames } from '../../classnames';
 import { Permission } from '../../query-runner/request/permissions';
 import { authenticationStyles } from '../Authentication.styles';
+import { hover } from '@testing-library/user-event/dist/types/convenience';
```

Comment:
Is this being used?

Reply:
That was fast :). Not really. Just removed it.
It was accidentally imported when I was initially adding hover behavior for the 'Get sandbox' link